Formula/jhiccup.rb

```diff
@@ -1,7 +1,7 @@
 class Jhiccup < Formula
   desc "Measure pauses and stalls of an app's Java runtime platform"
-  homepage "http://www.jhiccup.com"
-  url "http://www.azulsystems.com/sites/default/files/images/jHiccup.1.3.7.zip"
+  homepage "http://www.jhiccup.com/"
```

Comment:
This is now a redirect to `https://www.azul.com/products/open-source-tools/jhiccup-performance-measurement-tool/`